samples/testapp/src/commonMain/kotlin/com/android/identity/testapp/App.kt

```diff
@@ -66,6 +68,7 @@ class App {
         // Get the name of the current screen
         val currentDestination = appDestinations.find {
             it.route == backStackEntry?.destination?.route
+                    || backStackEntry?.destination?.route?.startsWith(it.route) == true // To support destinations with arguments.
```

Comment:
Btw, curious why this is needed? We already supports destinations with arguments but maybe that was broken?